### ISSUE TITLE
Can we move this project into JazzBand for better shared responsibility for maintenance?

### ISSUE THREAD

**Issue by philipsd6** [RidersDiscountCom/HypChat#61]
There are PRs that have sat open for 1.5 years, and Issues that have been left open 3 years. @successamp, you've said that Python isn't your strongest language (I'm no Python guru either) so would you consider supporting a plan to move this repository into https://jazzband.co/ so the community can help maintain it?

If you are ok with this, I would be happy to start the process meeting the standards required for JazzBand...

**Comment by wrhansen** [comment]
@philipsd6 We don't support this API anymore internally and let it get out of date. Let me know what you need to make this happen.

**Comment by philipsd6** [comment]
Thanks! I'm kind of a fan of this module -- I've emulated the API abstraction wrapper style in some other libraries I've used, and it works well. I'll submit a PR with some required documentation updates, and propose it be moved into the Jazzband team, but someone with admin access to this repo will need to approve it. I'll keep you posted.

**Comment by ridersdiscount** [comment]
Sounds good. Just let me know what you need done on our side.

**Comment by ridersdiscount** [comment]
@philipsd6 Did you still want to take control of this?

**Comment by philipsd6** [comment]
Yes, I do -- sorry it's taking me so long. I have a local branch with some of the work done. I'll be able to get back to this in about a month.
